app/src/main/java/org/ea/sqrl/activites/base/LoginBaseActivity.java

```diff
@@ -211,9 +214,23 @@ public void afterTextChanged(Editable s) {
             long currentId = sharedPref.getLong(CURRENT_ID, 0);
 
             if(currentId != 0) {
+                if(LoginBaseActivity.this instanceof EnableQuickPassActivity) {
+                    ((EnableQuickPassActivity)LoginBaseActivity.this).isDoingLogin();
+                }
                 doLogin(storage, txtLoginPassword, false, false, null, this);
             }
         });
+
```

Comment:
This should have been remove...it's only logging.  This is called if back pressed or if a button is pressed, so wasn't useful.

Reply:
It's fixed in 
dd29477